src/state/trajectory/constants.ts

```diff
@@ -35,6 +35,8 @@ export const RECEIVE_FILE_TO_CONVERT = makeTrajectoryConstant(
     "receive-file-to-convert"
 );
 export const CONVERT_FILE = makeTrajectoryConstant("convert-file");
+export const RECEIVE_CONVERTED_FILE =
+    makeTrajectoryConstant("receive-converted");
 
 export const SET_CONVERSION_STATUS = makeTrajectoryConstant("set-status");
 export const CONVERSION_INACTIVE = "conversion-inactive";
```

Comment:
why isn't this using the makeTrajectoryConstant function?

Reply:
I don't know! It should be.

Reply:
Oh I remember, these are n't action types they aren't specificlly used in redux, they're just used in conditional checks in the application related to conversion. I don't know that they should use `makeTrajectoryConstant` but maybe they should be declared in a different file?

Reply:
Either way they are going to be changed when we add the enum in the forthcoming PR...